src/.nuget/Microsoft.DotNet.CoreCLR.Debug.Development.nuspec

```diff
@@ -24,7 +24,7 @@
     <file src="..\corerun.exe" target="lib\aspnetcore50\corerun.exe" />
     <file src="..\mscorlib.dll" target="lib\aspnetcore50\mscorlib.dll" />
     <file src="..\sos.dll" target="lib\aspnetcore50\sos.dll" />
-    <file src="..\protojit.dll" target="lib\aspnetcore50\protojit.dll" />
+    <file src="..\ryujit.dll" target="lib\aspnetcore50\ryujit.dll" />
```

Comment:
We should delete ryujit.dll from the CoreCLR packages since the standalone version is its own package now, and CoreCLR has own version linked in.